### PR TITLE
[Docs]zh-cn document translate: keycode list page

### PR DESCRIPTION
**Pull request by epiciskandar**
* keycode list page

## Description

following after PR #14924

**notice: as described in the first part（Chinese annotation/译注部分）, the 'Description' parts of some tables didn't being translated, because it is better be done in those linked detailed feature pages. and also is good for keeping translation tracking and synchonizing.** 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* tracking issue: #15443

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/zh-cn/keycodes.md [hunk 1]
```diff
@@ -0,0 +1,780 @@
+# 键码概览
+
```
Comment (peepeetee) [inline docs/zh-cn/keycodes.md:2]:
```suggestion

<!---
  original document: 0.14.23:docs/keycodes.md
  git diff 0.14.23 HEAD -- docs/keycodes.md | cat
-->

```

**Review by peepeetee — CHANGES_REQUESTED** [review]

File: docs/zh-cn/keycodes.md [hunk 2]
```diff
@@ -0,0 +1,785 @@
...
+<!---
+  original document: 0.14.23:docs/keycodes.md
+  git diff 0.14.23 HEAD -- docs/keycodes.md | cat
```
Comment (daskygit) [inline docs/zh-cn/keycodes.md:5]:
```suggestion
  original document: 0.15.18:docs/keycodes.md
  git diff 0.15.18 HEAD -- docs/keycodes.md | cat
```

File: docs/zh-cn/keycodes.md [hunk 3]
```diff
@@ -0,0 +1,785 @@
...
+|`MAGIC_UNHOST_NKRO`               |`NK_OFF` |Disable N-key rollover                                                    |
+|`MAGIC_TOGGLE_NKRO`               |`NK_TOGG`|Toggle N-key rollover                                                     |
+|`MAGIC_EE_HANDS_LEFT`             |`EH_LEFT`|Set the master half of a split keyboard as the left hand (for `EE_HANDS`) |
+|`MAGIC_EE_HANDS_RIGHT`            |`EH_RGHT`|Set the master half of a split keyboard as the right hand (for `EE_HANDS`)|
+
+## MIDI :id=midi
+
+参见：[MIDI](zh-cn/feature_midi.md)
+
+（译注：TODO。由于翻译者对乐理基础掌握不到位，为确保不出现错误，暂不翻译。欢迎有能力的同志贡献一份力量。professional translating is welcome）
```
Comment (daskygit) [inline docs/zh-cn/keycodes.md:370]:
```suggestion

```

I'm not sure this is necessary? Maybe someone else could offer some input here or a translation of below?

**Review by daskygit — CHANGES_REQUESTED** [review]

File: docs/zh-cn/keycodes.md (hunk 3 above)
Comment (epiciskandar) [reply docs/zh-cn/keycodes.md:370]: myself actually not 100% confidental at this, if so we have to wait, I would try to find somebody helping at this.

**Review by peepeetee — APPROVED** [review]

**Comment by epiciskandar** [comment]
keep this

**Pull request closed without merging**